core/src/main/java/me/mykindos/betterpvp/core/effects/types/negative/ConcussedEffect.java

```diff
@@ -0,0 +1,26 @@
+package me.mykindos.betterpvp.core.effects.types.negative;
+
+import me.mykindos.betterpvp.core.effects.VanillaEffectType;
+import org.bukkit.potion.PotionEffectType;
+
+public class ConcussedEffect extends VanillaEffectType {
+
+    @Override
+    public String getName() {
+        return "Concussed";
+    }
+
+    @Override
+    public boolean isNegative() {
+        return true;
+    }
+
+    @Override
+    public PotionEffectType getVanillaPotionType() {
+        return PotionEffectType.SLOW_DIGGING;
+    }
+
+    public static double getAttackSpeedModifier() {
```

Comment:
This doesn't need to be static, you can already access the object via EffectTypes.CONCUSSED.getAttackSpeedModifier()

However I think this function is unnecessary, just do the calculation in the CombatListener

`  event.setDamageDelay((long) (event.getDamageDelay() * 1 + (effect.getAmplifier() * 0.25));`

Reply:
Changed in [3e81a4e](https://github.com/Mykindos/BetterPvP/pull/851/commits/3e81a4ef758bfb572871f7e9c606860308388f5d), "Added a sound effect that plays when a concussed player attacks; unhardcoded setDamageDelay value into a formula."